pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
+execution_timeout = 300
```

Comment:
increase this timeout to 600 (due to some algotest)

Reply:
done